### ISSUE TITLE
Question regarding Imagenet pretraining

### ISSUE THREAD

**Issue by MLDeS** [Sense-X/UniFormer#122]
Thanks for the nice work! I have a question regarding model training reported in the paper. Its says

> With only ImageNet-1K pretraining,
our UniFormer achieves 82.9%/84.8% top-1 accuracy on Kinetics-400/Kinetics600,

My question is the models are video models with n frames as input, whereas ImageNet is an image data with single inputs. So, my question is which part has ImageNet pretrained weights?

**Comment by Andy1621** [comment]
All the parts have ImageNet-pretraining. For convolution, if the temporal dimension is larger than 1, we will copy and average the convolution weights. For self-attention, we copy the same weights. Please check the code
https://github.com/Sense-X/UniFormer/blob/f92e423f7360b0026b83362311a4d85e448264d7/video_classification/slowfast/models/uniformer.py#L387-L421

**Comment by Andy1621** [comment]
For convolution inflation, I suggest you reading paper I3D.

**Comment by MLDeS** [comment]
Thanks a lot for the answers!
